requirements.txt

```diff
@@ -1,3 +1,5 @@
 pydiscourse>=1.3,<1.4
 PyYAML>=6.0,<6.1
 requests>=2.28,<2.29
+GitPython>=3.1,<3.2
```

Comment:
Can you order the deps alphabetically? It will be easier to track what's in the file as it grows

Reply:
Sounds like a good idea! Done :)